sphinx_wagtail_theme/layout.html

```diff
@@ -89,7 +89,7 @@
                             {%- if theme_logo_height %} height="{{ theme_logo_height|e }}"{% endif %}
                             {%- if theme_logo_alt %} alt="{{ theme_logo_alt|e }}"{% endif %}
                             {%- if theme_logo_title %} title="{{ theme_logo_title|e }}"{% endif %}
-                            class="logo-img"
+                            class="logo-img" decoding="async" loading="lazy"
```

Comment:
nitpick - please put these new attributes on their own lines - as all other attributes are on their own lines.

Also - maybe this should show instantly as it's the primary logo for the docs.

Reply:
sure!